app.json

```diff
@@ -19,6 +19,18 @@
       "generator": "secret"
     }
   },
+  "addons":[
+      "heroku-postgresql:hobby-dev",
+      "heroku-redis:hobby-dev"
+  ],
+  "buildpacks":[
+      {
+          "url":"heroku/nodejs"
+      },
+      {
+          "url":"heroku/python"
```

Comment:
Could you swap the order of the build packs ? Python first and nodejs next. I'll update the docs.